HW5/models/userModel.js

```diff
@@ -0,0 +1,73 @@
+const { v4: uuidv4 } = require("uuid");
+const path = require("path");
+
+const users = require("../data/users.json");
```

Comment:
Unfortunately, when you update a file, it doesn't update imported objects
Thus, when you add a new user, it isn't returned from getAllUsers method without restarting the server

Reply:
Thank you, updated